src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -1065,169 +1065,174 @@ public static int ConvertToUtf32(string s, int index)
         // IAdditionOperators
         //
 
-        static char IAdditionOperators<char, char, char>.operator +(char left, char right)
-            => (char)(left + right);
+        /// <inheritdoc cref="IAdditionOperators{TSelf, TOther, TResult}.op_Addition(TSelf, TOther)" />
+        static char IAdditionOperators<char, char, char>.operator +(char left, char right) => (char) (left + right);
 
-        // static checked char IAdditionOperators<char, char, char>.operator +(char left, char right)
-        //     => checked((char)(left + right));
+        // /// <inheritdoc cref="IAdditionOperators{TSelf, TOther, TResult}.op_Addition(TSelf, TOther)" />
+        // static char IAdditionOperators<char, char, char>.operator checked +(char left, char right) => checked((char)(left + right));
 
         //
         // IAdditiveIdentity
         //
 
+        /// <inheritdoc cref="IAdditiveIdentity{TSelf, TResult}.AdditiveIdentity" />
         static char IAdditiveIdentity<char, char>.AdditiveIdentity => (char)0;
```

Comment:
API review will need to discuss if exposing `AdditiveIdentity`, `MultiplicativeIdentity`, `One`, and `Zero` make sense for `char`. They are not exposed until that discussion happens.

All of the operators exist in regular C#, but unlike the other types the members being exposed publicly may be potentially confusing or "incorrect" terminology for `char`.